tests/amount/test_automapper_amount.py

```diff
@@ -13,8 +13,8 @@ def test_auto_mapper_amount(spark_session: SparkSession) -> None:
     # Arrange
     spark_session.createDataFrame(
         [
-            (1, 'Qureshi', 'Imran', "54.45"),
-            (2, 'Vidal', 'Michael', "67.67"),
+            (1, 'Qureshi', 'Imran', "54.45"), (2, 'Vidal', 'Michael', "67.67"),
+            (3, 'Alex', 'Hearn', "1286782.17")
         ], ['member_id', 'last_name', 'first_name', "my_age"]
```

Comment:
interesting "my_age"s :D

Reply:
(I didn't name the columns, just putting in data in the one I added that demonstrates the effect.)